pkg/destroy/aws/suppresswarning.go

```diff
@@ -0,0 +1,25 @@
+package aws
+
+import (
+	"time"
+
+	"github.com/sirupsen/logrus"
+)
+
+var (
+	errorTracker = map[string]time.Time{}
+)
+
+func suppressWarning(err error, identifier string, duration time.Duration, logger logrus.FieldLogger) {
```

Comment:
i think the duration should be const and not an argument.

Reply:
done.